MQTT_server.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+The structure of MQTT server consists of three components. `NB-IoT client`, `tangle-accelerator` and `Parser`
```

Comment:
Don't mention `NB-IoT` here. It should be an abstract term, such as connectivity endpoint.